scalafmt-tests/src/test/resources/rewrite/AvoidInfix.stat

```diff
@@ -388,7 +388,7 @@ object a {
 }
 >>>
 object a {
-  (a(b("c")).over(d) / (e + 1)).as(f)
+  ((a(b("c")).over(d)) / (e + 1)).as(f)
```

Comment:
Should the default behaviour change?

Reply:
actually, it's simply reverting back to pre-3.5.3 behavior; same with the other case.